.github/workflows/ci.yml

```diff
@@ -127,19 +111,10 @@ jobs:
 
     - uses: actions/checkout@v3
 
-    - uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af # v1.0.7
-      with:
-        profile: minimal
-        toolchain: stable
-        override: true
-        components: clippy
-
     - uses: Swatinem/rust-cache@6720f05bc48b77f96918929a9019fb2203ff71f8 # v2.0.0
 
-    - name: Run cargo clippy
-      uses: actions-rs/cargo@844f36862e911db73fe0815f00a4a2602c279505 # v1.0.3
-      with:
-        command: custom-clippy # cargo alias to allow reuse of config locally
```

Comment:
We are running an alias here, please retain that (with the comment!).

Reply:
But why are you running an alias here? I don't quite understand the assignment.

Reply:
The alias is a workaround for clippy still not having a config file.

It is defined here: https://github.com/libp2p/rust-libp2p/blob/master/.cargo/config.toml#L3

This allows us to allow certain lints across the codebase without having to duplicate this in a bunch of places :)